frontend/packages/kubevirt-plugin/src/components/modals/disk-modal/disk-modal.tsx

```diff
@@ -75,8 +75,10 @@ export const DiskModal = withHandlePromise((props: DiskModalProps) => {
   });
   const combinedDiskSize = combinedDisk.getSize();
 
+  const type = disk.getType() || DiskType.DISK;
+
   const [source, setSource] = React.useState<StorageUISource>(
-    isEditing ? combinedDisk.getSource() || StorageUISource.OTHER : StorageUISource.BLANK,
+    isEditing ? combinedDisk.getSource() || StorageUISource.URL : StorageUISource.BLANK,
```

Comment:
if the modal is opened with creating a New CD, `OTHER` is one of the unsupported sources, so the select element is disabled and the user can't do anything. are you against this?

Reply:
The reason for introducing OTHER is that we want to edit name and bus for disks with unsupported source in disks tab

This change concerns only editing - not new CDs. 

The defaults should be dependent on the type if you want different default for new CD (other than BLANK).

Reply:
I see that the isEditing detection is broken now. IMO we should add a prop isEditing to be clear and not depend on the disk value

Reply:
I fear this is out of scope for this PR, which is big anyway, agreed?

Reply:
Not out of scope IMO - this is the implementation of this PR. 

I don't think it is such a big change that it would justify introducing such bugs into the UI.